00-intro.md

```diff
@@ -11,15 +11,15 @@ minutes: 5
 
 At a high level, computers do four things:
 
--   run programs
--   store data
--   communicate with each other
--   interact with us
+-   run programs,
+-   store data,
+-   communicate with each other,
+-   and interact with us.
```

Comment:
It depends. In exercises usually no punctuation is used (but it makes sense in MCQs with lists of possible commands). On the other hand, learning objectives use punctuation. There is no more bullet list of the style above in the whole lesson. 

I tend to treat the bullets as part of the sentence, so the sentence should be grammatically correct if we remove the bullets and put everything on a single line. I think that is the rationale behind the cited style rule.

If this is the only block to merge this PR, I am happy to remove it (and maybe do another PR with the change above). Let me know what you recommend. Thanks!